### PR TITLE
Pull in upstream changes from cisagov/skeleton-ansible-role

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request pulls in upstream changes from [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role).

## 💭 Motivation and Context

I thought I needed to update this role to handle the Kali care explicitly, but I didn't.  While I was here, though, I figured I may as well pull in the latest and greatest upstream changes.

## 🧪 Testing

All pre-commit hooks and molecule tests pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: molecule/default/molecule.yml
Comment (dav3r) [inline molecule/default/molecule.yml:22]: Shouldn't Kali alphabetically go after Fedora and before Ubuntu?

**Review by dav3r — APPROVED** [review]
All good, though I had one question- see below.

**Review by mcdonnnj — APPROVED** [review]
Makes sense ✅

**Review by felddy — APPROVED** [review]
Looks good.  You should try to pull in our latest `CODEOWNERS` changes.

File: molecule/default/molecule.yml
Comment (jsf9k) [reply molecule/default/molecule.yml:22]: Yes, but @mcdonnnj did it this way in the skeleton; I assume this is because Kali is in the Debian family.  I decided that sorting by OS family and then sorting alphabetically within OS families was also reasonable.

If you want to put comments separating the OS families in this file, please do so in the skeleton.

**Comment by jsf9k** [comment]
> Looks good. You should try to pull in our latest `CODEOWNERS` changes.

Done!

File: molecule/default/molecule.yml
Comment (dav3r) [reply molecule/default/molecule.yml:22]: I didn't notice this in the skeleton- as long as we are doing it consistently, that is good enough for me.

**Review by felddy — APPROVED** [review]

**Review by jsf9k — COMMENTED** [review]
Nice work @mcdonnnj.  I didn't think to apply the split scenario foo here. :1st_place_medal: 

Go ahead and merge this when you're happy with it.

**Review by mcdonnnj — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
Hot stuff! 🔥

**Pull request merged**
